src/AspNet.Security.OAuth.QQ/QQAuthenticationDefaults.cs

```diff
@@ -14,21 +15,26 @@ namespace AspNet.Security.OAuth.QQ
     public static class QQAuthenticationDefaults
     {
         /// <summary>
-        /// Default value for <see cref="AuthenticationOptions.AuthenticationScheme"/>.
+        /// Default value for <see cref="AuthenticationScheme.Name"/>.
         /// </summary>
         public const string AuthenticationScheme = "QQ";
 
         /// <summary>
-        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+        /// Default value for <see cref="AuthenticationScheme.DisplayName"/>.
         /// </summary>
-        public const string CallbackPath = "/signin-qq";
+        public const string DisplayName = "QQ";
 
         /// <summary>
-        /// Default value for <see cref="AuthenticationOptions.ClaimsIssuer"/>.
+        /// Default value for <see cref="AuthenticationSchemeOptions.ClaimsIssuer"/>.
         /// </summary>
         public const string Issuer = "QQ";
 
         /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
```

Comment:
Please remove this arbitrary reordering change.

Reply:
This is not a reordering change, I added `DisplayName` field, but the diff result doesn't show this correctly.